odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/mapper/ToFhirMapping.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright © 2024, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.odoo.openmrs.mapper;
+
+import org.hl7.fhir.r4.model.Resource;
+
+/**
+ * An Interface for mapping from FHIR Resources to Odoo Documents
+ *
+ * @param <F> FHIR Resource
+ * @param <E> Odoo Document
+ */
+public interface ToFhirMapping<F extends Resource, E> {
+
+    /**
+     * Maps an Odoo Document to a FHIR Resource
```

Comment:
Document or Resource?

Reply:
I think I took it from ERPNext. I think in ERPNext a model is called a document. Let me rename it